hotsos/defs/scenarios/vault/bugs/lp1940549.yaml

```diff
@@ -0,0 +1,17 @@
+checks:
+  has_lp1940549:
+    snap:
+      vault:
+        # Looking for version less than 1.7
+        - min: 1967
+          max: 2039
+conclusions:
+  lp1940549:
+    decision:
+      - has_lp1940549
+    raises:
+      type: LaunchpadBug
+      bug-id: 1940549
+      message: >-
+        Vault may not renewed or reissued certificates. This may lead to other services being degraded due to TLS issues.
```

Comment:
The issue described in LP 1940549 is about the fact that the vault charm sometimes does not propagate certificates on charm relations correctly but is not a bug in vault itself. I do not feel that this message accurately depicts that and I'm also not sure that there is a way to accurate detect that the problem as occurred from a sosreport since the issue manifests on the remote side i.e. other charms that consume the certs.

Reply:
Also fwiw, you would need to squash your commit into a single commit.